src/configurations/destinations/tune/schema.json

```
@@ -0,0 +1,913 @@
+{
+  "configSchema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "required": [],
+    "type": "object",
+    "properties": {
+      "url": {
+        "type": "string",
+        "pattern": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|(?!.*\\.ngrok\\.io)^(?:http(s)?:\\/\\/)?[\\w.-]+(?:\\.[\\w\\.-]+)+[\\w\\-\\._~:/?#[\\]@!\\$&'\\(\\)\\*\\+,;=.]+$|^$"
+      },
+      "eventName": {
+        "type": "string",
+        "pattern": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{0,100})$"
+      },
+      "standardMapping": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "from": {
+              "type": "string",
+              "pattern": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{0,100})$"
+            },
+            "to": {
+              "type": "string",
+              "enum": [
+                "aff_id",
+                "promo_code",
+                "amount",
+                "conversion_unique_id",
+                "goal_id",
+                "goal_ref",
+                "offer_id",
+                "payout",
+                "revenue",
+                "security_token",
+                "status",
+                "transaction_id",
+                ""
+              ]
+            }
+          }
+        }
+      },
+      "advSubIdMapping": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "from": {
+              "type": "string",
+              "pattern": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{0,100})$"
+            },
+            "to": {
+              "type": "string",
+              "enum": ["adv_sub", "adv_sub2", "adv_sub3", "adv_sub4", "adv_sub5"]
```

Ignore if not applicable.
```suggestion
              "enum": ["adv_sub1", "adv_sub2", "adv_sub3", "adv_sub4", "adv_sub5"]
```

---

Yes, These properties are as per Tune's [doc](https://support.tune.com/hc/en-us/articles/9516563974167-Conversion-URL-Parameters).